### ISSUE TITLE
sql: Scan error on column index 2, name "is_primary"

### ISSUE THREAD

**Issue by fillon** [KarnerTh/mermerd#24]
Hi,

Just installed mermerd, look like a nice tool.

I tried to run it against a mysql database and get the following error
- mermerd 0.4.0
- macos amd

```
INFO[0006] Got tables                                    count=38
[=>                  ]Getting columns and constraintsERRO[0008] Getting columns failed | sql: Scan error on column index 2, name "is_primary": sql/driver: couldn't convert 2 into type bool
ERRO[0008] sql: Scan error on column index 2, name "is_primary": sql/driver: couldn't convert 2 into type bool


X Something went wrong!
```

Any idea ?  :-)

**Comment by KarnerTh** [comment]
Thanks for pointing out 👍

Yeah I have good guess what could be wrong, I will look into this

**Comment by ashutosrath** [comment]
I got also a similar issue 
```
INFO[0000] Got tables                                    count=13
? Choose tables: clouds, clusters
[=>                  ]Getting columns and constraintsERRO[0009] Getting columns failed | sql: Scan error on column index 3, name "is_foreign": sql/driver: couldn't convert 3 into type bool
ERRO[0009] sql: Scan error on column index 3, name "is_foreign": sql/driver: couldn't convert 3 into type bool

```

**Comment by KarnerTh** [comment]
@ashutosrath thanks for the PR 👍
I will release a new version in the near future

**Comment by KarnerTh** [comment]
The fix is now available in Version [0.4.1](https://github.com/KarnerTh/mermerd/releases/tag/v0.4.1)
